stack

```diff
@@ -53,6 +54,14 @@ function load-local-env() {
   done < .local.dev
 }
 
+############################################################################
+# Get build version info
+############################################################################
+function get-build-version() {
+  export VERSION=$(git rev-parse --abbrev-ref HEAD)
```

Comment:
It looks like this command returns the branch name:

```
»  git rev-parse --abbrev-ref HEAD
walkah/fix-add-version-to-docker-build
```

Perhaps we could use `git describe --tags --abbrev=0` to retrieve the latest tag? It won't be accurate for builds we push to `main` that aren't part of a release though.

Reply:
Or possibly `git tag --points-at HEAD`. It returns a list of tags though, not sure if the output is ideal.

Reply:
@bgins in the `release_docker` workflow, we're passing in the release tag name for VERSION. What's in `stack` would be intended for local dev, where I thought branch name made more sense. Does that help?

Reply:
Yep, it does! Thanks for the explanation.

When we run `release_docker` on push to `main`, this will show the version as `main` right? I'd assume resource providers generally won't be running these images, but it's a consideration for parsing the version when we check it.

Reply:
That's right, main will have a `main` version, with the corresponding git hash. Local docker builds will have branch name (plus git hash) and tagged releases will have tag / release name and git hash.